app/src/main/java/com/incwelltechnology/lms/ui/auth/AuthListener.kt

```diff
@@ -0,0 +1,9 @@
+package com.incwelltechnology.lms.ui.auth
+
+import androidx.lifecycle.LiveData
+
+interface AuthListener {
```

Comment:
@iamsannyrai do you always get the string as the response?

Reply:
No, response can be anything. I have input string as response to display common yet suitable plain English message to user who doesn't concern much about technical shortcomings